File: prak2 java/Product.java
class Product{

    private int price;
    private String idProduct;

    Product(){
        //konstruktor
    }

    Product(int price, String idProduct){
        //konstruktor
        this.price = price;
        this.idProduct = idProduct;
    }

    //get set
    public int getPrice() {
        return price;
    }

    public void setPrice(int price) {
        this.price = price;
    }

    public String getIdProduct() {
        return idProduct;
    }

    public void setIdProduct(String idProduct) {
        this.idProduct = idProduct;
    }

    //methods
    public void tampilkanProduct(){
        System.out.println("Price: " + getPrice());
        System.out.println("idProduct: " + getIdProduct());
    }

}

File: prak2 php/Product.php
<?php

class Product{

    private $price;
    private $idProduct;

    function __construct($pr, $id){
        //konstruktor
        $this->price = $pr;
        $this->idProduct = $id;
    }


    //set
    public function setPrice($pr){
        $this->price = $pr;
    }

    public function setIdProduct($id){
        $this->idProduct = $id;
    }

    //get
    public function getPrice(){
        return $this->price;
    }

    public function getIdProduct(){
        return $this->idProduct;
    }

    
    //methods
    public function tampilkanProduct(){
        echo "Price : ".$this->getPrice()."<br/>";
        echo "idProduct : ".$this->getIdProduct()."<br/>";
    }


    function __destruct(){
        //destruktor
    }

}

?>
